### ISSUE TITLE
Sparsity Experiment

### ISSUE THREAD

**Issue by robinhesse** [suinleelab/attributionpriors#4]
Hi,

Thanks a lot for this awesome project! I am currently trying to reproduce the results from the sparsity experiment and I think I found some minor bugs:

1) In `run_sparsity_experiments.py` in `line 277-289` (training the unregularized model) the `refset` must be enlarged to work with `REF=100` samples.
--> change line 281 to: 
`refset = torch.utils.data.TensorDataset(torch.cat([Xtrain_tensor]*REFS))`
Additionally, ytest is not defined so one needs to add 
`ytest = ytest_tensor.detach().cpu().numpy()`
after line 280

2) In `plot_results.ipynb` you are plotting `testscores`, `testsparsity`, and `testshaps` but I think none of them is defined. Since I am not sure how those values are computed/obtained it would be great if you could help me out.

Thank you very much & Best,
Robin
